packages/valory/skills/safe_deployment_abci/tests/test_behaviours.py

```diff
@@ -303,7 +303,7 @@ class TestValidateSafeBehaviour(BaseValidateBehaviourTest, SafeDeploymentAbciBas
     """Test ValidateSafeBehaviour."""
 
     behaviour_class = ValidateSafeBehaviour
-    next_behaviour_class = make_degenerate_behaviour(FinishedSafeRound.round_id)
+    next_behaviour_class = make_degenerate_behaviour(FinishedSafeRound.auto_round_id())
```

Comment:
When no longer specifying behaviour ids we need to use the class method to retrieve the auto generated id